src/common/catalog-entities/kubernetes-cluster.ts

```diff
@@ -132,6 +132,14 @@ export class KubernetesCluster<
         break;
     }
   }
+
+  get k8sVersion() {
+    return this.metadata.kubeVersion?.replace("unknown", "") || "";
```

Comment:
Do you actually want to replace the first occurrence of `unknown` or do you want to just want to return `""` instead of `"unknown"`?